spec/factories/kudos.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :kudo do
+    title { 'factory bot test title' }
+    content { 'factory bot test content' }
```

Comment:
I will suggest here for using Faker gem, as it is nice to have every next kudo with different title and content. Or instead of using Faker gem you can use sequence to add numbers in title and content